src/server/project.ts

```diff
@@ -616,22 +616,24 @@ namespace ts.server {
 
                 // Files that are no longer missing (e.g. because they are no longer required)
                 // should no longer be watched.
-                for (const missingFilePath of this.missingFilesMap.getKeys())
+                const missingFilesToDelete: string[] = [];

```

Comment:
It should be safe to delete values from a map while iterating over it. Are you sure this was the cause of an error?